api/src/main/java/org/apache/iceberg/PartitionField.java

```diff
@@ -60,7 +69,7 @@ public String name() {
 
   @Override
   public String toString() {
-    return name + ": " + transform + "(" + sourceId + ")";
+    return name + " (" + fieldId + "): " + transform + "(" + sourceId + ")";
```

Comment:
Nit: schemas use the format `1000: ts_day`. That would be a bit cleaner than two sets of parens.

Reply:
👌